tests/system/python/conftest.py

```diff
@@ -212,6 +214,156 @@ def clone_make_install():
 
     return _add_fledge_north
 
+@pytest.fixture
+def add_service():
+    def _add_service(fledge_url, service, service_branch, retries, installation_type = "make", service_name = "play",
+                     use_pip_cache = True, enabled = True):
+        
+        """ 
+            Fixture to add Service and start the start service by default 
+            service: Service to be installed
+            service_branch: Branch of service to be installed
+            retries: Number of tries for polling
+            installation_type: Type of installation for service i.e. make or package
+            serice_name: Name that will be given to service to be installed
+            use_pip_cache: use pip cache in requirements installation
+            enabled: Flag to enable or disable notification instace
+        """
+        
+        retval = utils.get_request(fledge_url, "/fledge/service")
+        for ele in retval["services"]:
+            if ele["type"].lower() == service:
+                return ele
+        
+        PROJECT_ROOT = Path(__file__).parent.parent.parent.parent
```

Comment:
This is a wrong practice to find the parent root.

Reply:
I know and I have found the new best way to find the project root i.e. `PROJECT_ROOT = subprocess.getoutput("git rev-parse --show-toplevel")`, but Praveen Sir Suggested me to keep old way of finding the PROJECT_ROOT and in other PR update all occurences of PROJECT_ROOT with `PROJECT_ROOT = subprocess.getoutput("git rev-parse --show-toplevel")`